src/device/device.c

```diff
@@ -53,7 +53,7 @@ void init_device(struct device* dev,
     uint16_t eeprom_id, struct storage_backend* eeprom_storage,
     struct clock_backend* clock,
     /* vi */
-    unsigned int vi_clock, unsigned int expected_refresh_rate, unsigned int count_per_scanline, unsigned int alternate_timing)
+    unsigned int vi_clock, unsigned int expected_refresh_rate, unsigned int count_per_scanline, unsigned int alternate_timing, unsigned int fixed_audio_pos)
```

Comment:
move fixed_audio_pos parameter upper in the parameter list be in the AI section please.